Events/2021-11-rse-hps-workshop.md

```diff
@@ -0,0 +1,54 @@
+# Research Software Engineers in HPC Workshop at SC22 (RSE-HPC-2022)
+- Date: July 11, 2022
```

Comment:
Is this supposed to be today's date, or the submission deadline (August 5)?

Reply:
Should we submission deadline. I am fixing this.